threescale_api/utils.py

```diff
@@ -64,6 +61,14 @@ def __init__(self, app, endpoint: str = "sandbox_endpoint",
 
         logger.debug("[HTTP CLIENT] New instance: %s", self._baseurl)
 
+    @staticmethod
+    def retry_for_session(session: requests.Session, total: int = 4):
+        retry = Retry(total=total, backoff_factor=1, status_forcelist=(503,), raise_on_status=True,
```

Comment:
raise_on_status should be rather False. this should be 'transparent' hence for the caller it should behave like it wasn't there (except of longer time due to retries)

Reply:
Fixed